.github/workflows/lte-integ-test-bazel.yml

```diff
@@ -0,0 +1,122 @@
+---
+
+name: LTE integ test bazel
+
+on:
+  workflow_dispatch: null
+  workflow_run:
+    workflows:
+      - build-all
+    branches:
+      - master
+      - 'v1.*'
+    types:
+      - completed
+
+jobs:
+  lte-integ-test-bazel:
+    if: github.repository_owner == 'magma' || github.event_name == 'workflow_dispatch'
+    runs-on: macos-10.15
+    env:
+      SHA: ${{ github.event.workflow_run.head_commit.id || github.sha }}
+    steps:
+      - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # pin@v2
+        with:
+          ref: ${{ env.SHA }}
```

Comment:
Why is this needed? Is that not the default behaviour to check-out the SHA of the triggering event?

Reply:
See updated PR description